cloudformation/template.yml

```diff
@@ -75,12 +71,25 @@ Parameters:
   Project:
     Type: String
     Default: Raster Vision CloudFormation
-    Description: Name of the project associated with the Batch job
+    Description: Name of the project to associate with new resources
 
   Environment:
     Type: String
     Default: Unknown
     Description: Environment of the project being deployed (e.g. Staging, Development, Production)
+    MinLength: 1
+    AllowedPattern: ^[a-zA-Z0-9]*$
+    ConstraintDescription: must only contain uppercase and lowercase letters and numbers
+
+  Slug:
```

Comment:
I would replace usage of the word "Slug" with "Prefix." I think that better represents what the variable is used for.

Reply:
Nice, that's much clearer.